Support/bone101/UI/bbui.js

```diff
@@ -909,6 +915,144 @@ var UI = (function() {
             return graph;
         })();
 
+        //graph global variables
```

Comment:
I'm adding a comment to remind myself to confirm the scope on these. We want to avoid/minimize global variables to make this easier to embed.